var/spack/repos/builtin/packages/py-meson-python/package.py

```diff
@@ -14,10 +14,7 @@ class PyMesonPython(PythonPackage):
 
     maintainers("eli-schwartz", "adamjstewart", "rgommers")
 
-    # Needed for py-scikit-image, deprecate and remove once 0.13.0 is officially released
-    version(
-        "0.13.0.pre0", sha256="8d537a0304709c31c11ffa34872a62a4c06a6a6c24fc862b7fb4306f3e881b95"
```

Comment:
Should prob be deprecated before removing but I'll leave that up to the other maintainers